fe/fe-core/src/main/java/org/apache/doris/alter/AlterJobV2.java

```diff
@@ -171,15 +186,28 @@ public synchronized void run() {
             return;
         }
 
+        // /api/debug_point/add/FE.ALTER_JOB_V2_RUN?stop=true
+        if (DebugPointUtil.isEnable("FE.ALTER_JOB_V2_RUN")) {
```

Comment:
boolean stop = DebugPointUtil.getDebugParamOrDefault(FE.ALTER_JOB_V2_RUN, "stop", false)

Reply:
use  if (DebugPointUtil.isEnable("FE.STOP_ALTER_JOB_RUN")) { return; }  ?

each point name should clear ?